docs/custom-flows/saml-connections.mdx

```diff
@@ -165,12 +163,10 @@ The following example shows how to handle these cases in your sign-in flow. The
         } else {
           // If the user has an account in your application
           // and has an SAML account connected to it, you can sign them in.
-          signInWith(e)
```

Comment:
`signInWith` was a bit vague here, this also seems to be copied from the OAuth connections code, where you would call like `signInWith('oauth_google')`